source/_patterns/00-protons/__tests__/protons.test.js

```diff
@@ -0,0 +1,7 @@
+import protons from '../';
+
+test('GLOBAL_CONSTANT of `blerp` is available', () => {
```

Comment:
Since these test files are likely to be copied and used as the starting place for new test files, should there be more comments? Boring stuff like 

```
/**
 * These tests are focused on protons.
 *
 * The tests below are only a start, and should be expanded with X, Y, and Z for more completeness.
 */
```

Reply:
I'll make a comment that `blerp` serves no other purpose than a starting point.